packages/ia-components/sandbox/audio-players/youtube-player.less

```diff
@@ -0,0 +1,14 @@
+.YoutubePlayer {
+  width: 99.5%;
+  height: 4.4rem; // accomodate height for jwplayer controls
+}
+
+.YoutubePlayer iframe {
+    position: absolute;
+    top: 0.1rem;
+    left: 0.1rem;
+    width: 100%;
```

Comment:
When you're setting a value for all four positioning properties, width and height don't have an effect on the element and can be omitted.

Reply:
Player expands out of the frame when height and width are omitted 😕

Reply:
Odd. So does it already have width and height property or attribute assignments that you're overriding?

Reply:
Yes, it's overriding the original height and width (600px) of the player.